package.json

```diff
@@ -101,6 +101,7 @@
     "@storybook/addons": "6.3.6",
     "@storybook/api": "6.3.6",
     "@storybook/builder-webpack5": "6.3.6",
+    "ignore-not-found-export-webpack-plugin": "1.0.1",
```

Comment:
It looks like updating to 1.0.2 will fix this. (Also please put it in alphabetical order.)
https://unpkg.com/browse/ignore-not-found-export-webpack-plugin@1.0.2/package.json